builder/scaleway/config.go

```diff
@@ -265,11 +271,19 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 	}
 
 	if c.ShutdownTimeout == "" {
-		c.ShutdownTimeout = "5m"
+		c.ShutdownTimeout = defaultInstanceServerWaitTimeout
```

Comment:
Just initialize the c fields with the default value you know are correct and then test that the configuration provided is not an empty string and that they pass the parsing. If all of that is ok, then override the value in the config with the value provided by the user.